documentation/src/main/asciidoc/ch11.asciidoc

```diff
@@ -694,17 +694,25 @@ mechanism to register additional constraint definitions. All you have to do is t
 _javax.validation.ConstraintValidator_ to _META-INF/services_. In this service file you list the
 fully qualified classnames of your constraint validator classes (one per line). Hibernate Validator
 will automatically infer the constraint types they apply to.
-See <<example-using-service-file-for-constraint-definitions>> for an example.
+See <<example-using-service-file-for-constraint-definitions,Constraint definition via service file>>
+for an example.
 
 [[example-using-service-file-for-constraint-definitions]]
-.Using _META-INF/services/javax.validation.ConstraintValidator_ for constraint definitions
+._META-INF/services/javax.validation.ConstraintValidator_
+
 ====
 [source]
 ----
+# Assuming a custom constraint annotation @org.mycompany.CheckCase
 org.mycompany.CheckCaseValidator
 ----
 ====
 
+To contribute default messages for your custom constraints, place a _ValidationMessages.properties_
```

Comment:
> What would that be?

E.g. `ConstraintValidationMessages`, `ProvidedValidationMessages`, `DefaultValidationMessages`, `HibernateValidatorValidationMessages`.

> Where is your concern about this?

Re-using the BV-defined name as is may lead people to the assumption that this is a portable feature, while it actually is HV-specific.

Also it might conflict in case BV one day actually supports `ValidationMessages` in other places than root. Depending on the then defined semantics, issues may arise upon upgrading. All in all I feel it's safer to stay away from the BV-defined name.